test/ScanForErrorMiddleware/ScannerTest.php

```diff
@@ -125,10 +139,13 @@ public function testScanningEmitsInfoToConsoleWhenEncounteringFilesOfInterest()
         $this->console
             ->writeLine(
                 Argument::that(function ($arg) {
-                    if (! strstr($arg, 'src/ErrorMiddleware.php')) {
+                    if (false === strpos($arg, 'src/ErrorMiddleware.php')) {
```

Comment:
As far as I know `strpos` is more efficient, uses less memory because it returns `int` or `false`. `strstr` returns portion of string  or `false`. It's just a good practice to use `strpos` instead (when possible).